Makefile

```diff
@@ -10,3 +10,29 @@ $(SUBDIRS):
 
 # Code generation
 include scripts/Makefile.codegen
+
+include scripts/inc.Makefile
+
+GOPKGS = ./pkg/adapter/... ./pkg/reconciler/...
```

Comment:
@antoineco should i change this to `./pkg/...` instead?

Reply:
Maybe just add `./pkg/apis/...`. Including `client/generated` would make your coverage really bad :D